From 90eed828652aff033fd07698fda75b51e1dbaf37 Mon Sep 17 00:00:00 2001
Message-Id: <90eed828652aff033fd07698fda75b51e1dbaf37.1701814941.git.shivajiva101@hotmail.com>
From: shivajiva101 <shivajiva101@hotmail.com>
Date: Tue, 5 Dec 2023 22:22:03 +0000
Subject: [PATCH] Removes argon2_cffi dependency

---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index dcc0d11b3..f6757ad43 100644
--- a/setup.py
+++ b/setup.py
@@ -39,7 +39,7 @@ bundled_plugins = [
     "OctoPrint-PiSupport>=2023.5.24",
 ]
 core_deps = [
-    "argon2_cffi>=21.3.0,<22",
+    #"argon2_cffi>=21.3.0,<22",
     "Babel>=2.12.1,<2.13",  # breaking changes can happen on minor version increases
     "cachelib>=0.10.2,<0.11",
     "Click>=8.1.3,<9",
--
2.39.2

